docs/en/ingest-management/fleet/fleet-server.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[fleet-server]]
+= {fleet-server}
+
+beta[]
+
+{fleet-server} is a component used to centrally manage {agent}s. It serves as a
+control plane for updating agent policies, collecting status information, and
+coordinating actions across agents.
+
+//QUESTION: I think it's worth breaking down in detail how data flows between
+// the agent, Fleet Server, Fleet, and the stack, but I might have some details
+// wrong here. Is this correct?
+
+{fleet-server} is the mechanism that connects {agent}s to {fleet} and handles
+all communication between an {agent}, {fleet}, {es}, and {kib}. {fleet-server}:
+
+* Receives status information from connected {agent}s
+* Retrieves agent policies from {fleet} and stores them in {es}
+* Ships agent policies to all connected {agent}s
+* Handles communication with {es} and {kib}
+
+image:images/fleet-server-communication.png[Fleet Server handles communication between {agent}, {fleet}, {es}, and {kib}]
+
+{fleet-server} runs as a separate service alongside {agent}. The agent uses a
+special policy that governs how {fleet-server} connects to other systems and
```

Comment:
other systems? The fleet-server integration policy describes the config options of the fleet-server itself.

Reply:
Sometimes we write fiction to get your attention. :-)